src/test/java/com/fasterxml/jackson/core/io/TestJsonStringEncoder.java

```diff
@@ -37,7 +50,24 @@ public void testQuoteLongAsString() throws Exception
         assertEquals(exp, new String(result));
         
     }
-    
+
+    // For [JACKSON-853]
```

Comment:
Could you remove this, or change to point to [core#266] (or whatever was id of the issue)?